lib/client-api/src/story_store.ts

```diff
@@ -416,41 +423,41 @@ export default class StoryStore {
       .map(({ id }) => this.fromId(id));
   }
 
-  extract(options?: StoryOptions) {
+  extract(options: StoryOptions & { normalizeParameters?: boolean } = {}) {
     const stories = Object.entries(this._stories);
-    // determine if we should apply a sort to the stories or use default import order
-    if (Object.values(this._stories).length > 0) {
-      const index = Object.keys(this._stories).find(
-        (key) =>
-          !!(
-            this._stories[key] &&
-            this._stories[key].parameters &&
-            this._stories[key].parameters.options
-          )
-      );
-      if (index && this._stories[index].parameters.options.storySort) {
-        const storySortParameter = this._stories[index].parameters.options.storySort;
-        let sortFn: Comparator<any>;
-        if (typeof storySortParameter === 'function') {
-          sortFn = storySortParameter;
-        } else {
-          sortFn = storySort(storySortParameter);
-        }
-        stable.inplace(stories, sortFn);
+
+    const storySortParameter = this._globalMetadata.parameters?.options?.storySort;
```

Comment:
NOTE: This means you can *only* set `storySort` via a global parameter. IMO it doesn't make sense any other way anyway but this is a breaking change.

Reply:
WFM

Reply:
And maybe throw an error if it gets set anywhere else?

Reply:
Good idea.